core/src/stack/inputs.rs

```diff
@@ -3,6 +3,11 @@ use crate::utils::{collections::*, ByteReader, Deserializable, DeserializationEr
 use super::{ByteWriter, Felt, InputError, Serializable, ToElements};
 use core::slice;
 
+// CONSTANTS
+// ================================================================================================
+
+const MAX_STACK_INPUTS_SIZE: usize = u16::MAX as usize;
```

Comment:
I would make this a public constant in the `StackInputs` impl. This way, it will be a part of the public interface.